app/javascript/components/application_form/preferences/AliceGriffithFields.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import { map } from 'lodash'
+import { Select, Text } from 'react-form'
+
+import { buildFieldId } from './utils'
+import { typeOfProofValues } from './values'
+import Row from '~/components/atoms/Row'
+import Column from '~/components/atoms/Column'
+import FormGroup from '~/components/atoms/FormGroup'
+
+const buildTypeOfProofOptions = (values) => {
+  return map(values, (option) => (
+    { value: option, label: option }
+  ))
+}
+
+const AddressRow = ({fieldId}) => (
+  <React.Fragment>
+    <Row form>
+      <Column span={6} form>
+        <FormGroup>
+          <label>Alice Griffith Address</label>
```

Comment:
Best practice would be to add `for` attribute to all form labels so they are connected to their associated input fields... unfortunately a lot of existing fields are missing this.